rasa_nlu/evaluate.py

```diff
@@ -714,7 +714,8 @@ def run_evaluation(data_path, model,
     else:
         interpreter = Interpreter.load(model, component_builder)
     test_data = training_data.load_data(data_path,
-                                        interpreter.model_metadata.language)
+                                        interpreter.model_metadata.language,
+                                        validate=False)
```

Comment:
I think we need to handle this for the crossvalidation functionality as well

Reply:
I didn't remove the validation there, since there is some actual training done and hence the warning actually is helpful. What you think?

Reply:
well, yes, but it splits them up into two seperate sets and as far as i know that gets registered in the logs. Is there no seperate function for the test data?

Reply:
The were no validations after the train test split for the cross validation. I now moved the validation to the training method. So there are now validations for the the cross validations but no more validations for evaluations.

Reply:
Hmmm ok now we're gonna have a log message for each pipeline that's trained though right?

Reply:
Nah, the pipeline stuff happens after:

```python
 def train(self, data: TrainingData, **kwargs: Any)-> 'Interpreter':
        """Trains the underlying pipeline using the provided training data."""

        self.training_data = data

        self.training_data.validate() # <-- validation

        context = kwargs

        for component in self.pipeline: # <-- pipeline stuff
            updates = component.provide_context()
            if updates:
                context.update(updates)

        # Before the training starts: check that all arguments are provided
        if not self.skip_validation:
            components.validate_arguments(self.pipeline, context)

        # data gets modified internally during the training - hence the copy
        working_data = copy.deepcopy(data)

        for i, component in enumerate(self.pipeline):
            logger.info("Starting to train component {}"
                        "".format(component.name))
            component.prepare_partial_processing(self.pipeline[:i], context)
            updates = component.train(working_data, self.config,
                                      **context)
            logger.info("Finished training component.")
            if updates:
                context.update(updates)

        return Interpreter(self.pipeline, context)

```

Reply:
ok sorry, clearly I was preoccupied with something else 😂